mazerunner/src/main/java/com/bugsnag/android/mazerunner/scenarios/HandledExceptionSessionScenario.kt

```diff
@@ -12,7 +13,7 @@ internal class HandledExceptionSessionScenario(config: Configuration,
 
     override fun run() {
         super.run()
-        disableSessionDelivery()
+        disableSessionDelivery(Bugsnag.getClient())
```

Comment:
In the future it would be helpful to separate the refactoring parts of this changeset (moving the internal functions to a separate file) into a separate commit. It wasn't initially clear whether these changes were otherwise related to the new scenario.